e2e/mobile/page/liveApps/swapLiveApp.ts

```diff
@@ -113,6 +116,26 @@ export default class SwapLiveAppPage {
     throw new Error("No providers without KYC found");
   }
 
+  @Step("Wait for quotes countdown to be stable")
+  async waitForQuotesStable(timeout: number = 20000) {
```

Comment:
Does it affect execution speed ?
Meaning, is the function quick enough ? Because I recall that getting web elements could sometimes be slow, if that's the case, could the retry often fail and increase testing time ?